### ISSUE TITLE
'pihole -v' seems to have issues..

### ISSUE THREAD

**Issue by fmjensen** [pi-hole/pi-hole#1523]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:** 

_{5}_

---
**[FEATURE REQUEST | QUESTION | OTHER]:**

Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[ISSUE] Expected Behaviour:**

```
me@debian:~$ pihole -v -h
:::
::: Show Pi-hole/Web Admin versions
:::
::: Usage: pihole -v [ -a | -p | -f ] [ -l | -c ]
:::
::: Options:
:::  -a, --admin      Show both current and latest versions of web admin
:::  -p, --pihole     Show both current and latest versions of Pi-hole core files
:::  -f, --ftl        Show both current and latest versions of  FTL engine <--- NEW OPTION
:::  -l, --latest     (Only after -a | -p **| -f** ) Return only latest version  <--- NEW OPTION
:::  -c, --current    (Only after -a | -p **| -f** ) Return only current version  <--- NEW OPTION
:::  -h, --help       Show this help dialog
:::
me@debian:~$ pihole -v
::: Pi-hole version is v3.0.1 (Latest version is v3.0.1)
::: Web-Admin version is v3.0.1 (Latest version is v3.0.1)
::: Pihole-FTL is v2.7.4 (Latest version is v2.8) <--- NEW OPTION
me@debian:~$ pihole -v -a
::: Web-Admin version is v3.0.1 (Latest version is v3.0.1)
me@debian:~$ pihole -v -p
::: Pi-hole version is v3.0.1 (Latest version is v3.0.1)
me@debian:~$ pihole -v -f <--- NEW OPTION
::: Pi-holeFTL version is v2.7.4 (Latest version is v2.8)  <--- NEW OPTION
me@debian:~$ pihole -v -a -p
::: Latest Web-Admin version is v3.0.1
me@debian:~$ pihole -v -a -c
::: Latest Pi-Hole version is v3.0.1
:~$ pihole -v -a -f <--- NEW OPTION
::: Latest Pi-HoleFTL version is v2.8 <--- NEW OPTION
me@debian:~$
```

**[ISSUE] Actual Behaviour:**
```
me@debian:~$ pihole -v -h
:::
::: Show Pi-hole/Web Admin versions
:::
::: Usage: pihole -v [ -a | -p ] [ -l | -c ]
:::
::: Options:
:::  -a, --admin          Show both current and latest versions of web admin
:::  -p, --pihole          Show both current and latest versions of Pi-hole core files
:::  -l, --latest          (Only after -a | -p) Return only latest version
:::  -c, --current          (Only after -a | -p) Return only current version
:::  -h, --help          Show this help dialog
:::
me@debian:~$ pihole -v
::: Pi-hole version is v3.0.1 (Latest version is v3.0.1)
::: Web-Admin version is v3.0.1 (Latest version is v3.0.1)
me@debian:~$ pihole -v -a
::: Invalid Option!  <-- ISSUE ??
me@debian:~$ pihole -v -p
::: Invalid Option!  <-- ISSUE ??
me@debian:~$ pihole -v -a -p
::: Invalid Option!  <-- ISSUE ??
me@debian:~$ pihole -v -a -c
v3.0.1  <-- ISSUE / incomplete ??
me@debian:~$
```

**[ISSUE] Steps to reproduce:**

- use the `pihole -v` command with various switches

**(Optional) Debug token generated by `pihole -d`:**

`<token>`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by DL6ER** [comment]
There is a major overhaul of the version script that will be shipped with Pi-hole v3.1

**Comment by WaLLy3K** [comment]
Issue resolved with release of v3.1.
